torch/package/importer.py

```diff
@@ -179,6 +185,14 @@ def get_source(self, module_name) -> str:
         module = self.import_module(module_name)
         return self.zip_reader.get_record(module.__file__).decode('utf-8')
 
+    def _install_on_parent(self, parent: str, name: str, module: types.ModuleType):
+        if not parent:
```

Comment:
does `not parent` mean `len(parent) == 0`? Or is `parent` actually `Option[str]`? Either way, would be good to make it explicit